<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手服务端</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1,1,0,2">
     <item>
      <widget class="QPlainTextEdit" name="editRev"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,2,0,1,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通信协议</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxProt">
         <item>
          <property name="text">
           <string>TCP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UDP</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>服务器IP地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxAddr"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPort">
         <property name="text">
          <string>8888</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnStartListen">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>开始监听</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopListen">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>停止监听</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOverListen">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MycomboBox" name="comboBoxChild"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="editSend">
         <property name="plainText">
          <string>test from server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MycomboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
